client/src/js/modules/shipment/views/containerplate.js

```diff
@@ -655,8 +655,13 @@ define(['marionette',
         updateClasses: function() {
             var first = this.autoscores.at(0)
             var opts = ''
+            var defaultClass = "crystal"
             _.each(first.get('CLASSES'), function(prob, cl) {
-                opts += '<option value="'+cl+'">'+cl+'</option>'
+                if (cl == defaultClass) {
```

Comment:
Is there a reason for introducing `defaultClass` rather than just comparing against `"crystal"`? Also, maybe it's a good idea to use `===` rather than `==`?

Reply:
I felt that explicitly naming it as the defaultClass helped show why it was there.

I'll change `==` to `===`.

Reply:
Okay, that makes sense